nrf_modem/doc/gnss_interface.rst

```diff
@@ -476,6 +476,30 @@ In cases where GNSS is not running continuously, it may be beneficial to change
 
    1PPS is only supported by modem firmware v1.3.0 or later.
 
+Resolving the UTC time of 1PPS pulse occurrence
+===============================================
+
+As the next pulse time (beginning of an UTC second) is calculated from the previous valid PVT fix, the latest PVT fix notification needs to be used when resolving the UTC time of the 1PPS pulse.
```

Comment:
```suggestion
As the time pulse (aligned to the top of an UTC second) is calculated from the previous valid PVT fix, the latest PVT fix notification needs to be used when resolving the UTC time of the 1PPS pulse.
```

Reply:
Or maybe "time of the pulse"?

Can you say "aligned to the **top** of a second"? @peknis?

Reply:
I have to say it sounds a bit weird. Can we somehow go around this?

Reply:
"aligned to the beginning of", "aligned to the boundary of". Any other ideas?

Reply:
This top of the second thing is basically reflecting u-blox timing parlance. I always assumed it is correct way to say it, but I don't know if I've actually seen it being used elesewhere.

Reply:
Okay, top of a second seems to be common expression in the domain of timing. If you google "top of a second" & UTC you'll find plenty of hits

Reply:
Seems that you are right, this just didn't ring a bell for me. @peknis, is "aligned to the top of an UTC second" then fine for you?

And do we go with "the time pulse"? To me "time of the pulse" sounds better, because that's what we're actually calculating.

Reply:
To be honest, I'm not really familiar with the topic. If you say the "top of a second" is a common expression, so be it. Same with the "time of the pulse".